Enable/View Controllers/ComposeViewController.m

```diff
@@ -313,4 +289,29 @@ - (void) setupTheme{
     [self.photosImageView setTintColor: [UIColor colorNamed: colorSet[@"Accent"]]];
 }
 
+- (void)picker:(nonnull PHPickerViewController *)picker didFinishPicking:(nonnull NSArray<PHPickerResult *> *)results {
+    int count = 0;
+    imageIndex = 0;
+    [self.images removeAllObjects];
+    for (PHPickerResult * res in results){
+        if([res.itemProvider canLoadObjectOfClass:[UIImage class]]){
+            [res.itemProvider loadObjectOfClass:[UIImage class] completionHandler:^(__kindof id<NSItemProviderReading>  _Nullable object, NSError * _Nullable error) {
+                if(object){
+                    dispatch_async(dispatch_get_main_queue(), ^{
+                        if(count == 0) {
+                            self.photosImageView.image = (UIImage*)object;
+                        }
+                        if(self.images.count < kMaxNumberOfImages){
```

Comment:
Ah got it, do we have some feedback if someone chooses more than max number?

Reply:
PHPicker makes it impossible for someone to choose more than the max; I mostly have this code here to safeguard